eggdrop-basic.conf

```diff
@@ -162,13 +162,14 @@ set prefer-ipv6 0
 
 ## See eggdrop.conf for documentation on the logfile flags and settings.
 
-## This creates a logfile named eggdrop.log containing private msgs/ctcps,
-## commands, errors, and misc. info from any channel.
+# This creates a logfile named eggdrop.log that captures items logged at the
+# m, c, and o log levels (private msgs/ctcps, commands/errors, and misc 
```

Comment:
At `putloglev` you change "level" to flag and here you introduce "level" again. Same below.